src/shared/standalone-date-input/standalone-date-input.tsx

```diff
@@ -318,7 +320,7 @@ export const Component = (
                     onBlur={handleInputBlur}
                     onChange={handleInputChange}
                     type="text"
-                    inputMode="numeric"
+                    inputMode={hideInputKeyboard ? "none" : "numeric"}
```

Comment:
Can specify a const `inputMode` so that you need not repeat the logic 3 times.

Reply:
Updated